# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../upower/glib_signal_fix.patch
# Copyright (C) 2012 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- upower-0.9.11/src/up-main.c.orig	2012-10-24 11:14:00.000000000 -0400
+++ upower-0.9.11/src/up-main.c	2012-10-24 11:14:22.000000000 -0400
@@ -203,7 +203,7 @@
 
 #if GLIB_CHECK_VERSION(2,29,4)
 	/* do stuff on ctrl-c */
-	g_unix_signal_add_watch_full (SIGINT,
+	g_unix_signal_add_full (SIGINT,
 				      G_PRIORITY_DEFAULT,
 				      up_main_sigint_cb,
 				      loop,
